shells.nix

```diff
@@ -58,7 +58,8 @@ in
                       nativeBuildInputs = oldAttrs.nativeBuildInputs or [ ]
                       ++ oldAttrs.passthru.shellInputs or [ ]
                       ++ oldAttrs.shellInputs or [ ]
-                      ++ [ shellCommands ];
+                      ++ [ shellCommands ]
+                      ++ lib.optional (oldAttrs ? targetSetup) oldAttrs.targetSetup;
```

Comment:
Is there a point (except backwards compatibility) to keep it as a separate drv attribute? Adding it to shellInputs is more intuitive.

Reply:
Backwards compatibility the only reason. But yes, I agree, for next major version we should probably change that. And I guess we could just make a patch release of where main is now and do it in this PR. I don't think we have any known bugs we want to get fixed before doing something that would require a lot of change.